core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -689,6 +689,18 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
     new ParallelCollectionRDD[T](this, seq, numSlices, Map[Int, Seq[String]]())
   }
 
+  /** Generate Long data from 0 until `limit` to form an RDD.
```

Comment:
javadoc style. and maybe say "Creates a new RDD[Long] containing elements from 0 to limit (exclusive)."